libs/utils/utils.js

```diff
@@ -424,6 +424,27 @@ export function decorateSVG(a) {
   }
 }
 
+function hasUrlWithPipe(str) {
+  const regex = /^https?:\/\/.*\|.*/;
+  return regex.test(str);
+}
+
+export function decorateImageLinks(a) {
+  const images = a.querySelectorAll('img');
+  if (!images.length) return a;
+  [...images].forEach((img) => {
+    if (!img.alt || !(hasUrlWithPipe(img.alt))) return;
+    const [source, alt] = img.alt.split('|');
+    const textUrl = new URL(source.trim());
+    if (alt && alt.trim().length) img.alt = alt.trim();
+    const pic = img.closest('picture');
+    const picParent = pic.parentElement;
+    const aTag = createTag('a', { href: textUrl, class: 'image-link' }, pic);
+    picParent.append(aTag);
```

Comment:
I tested this out and it looks like the `const picParent` does needs to be declared otherwise the pic.parentElement cant append to itself and throws an error.